.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I added this config here to fix the issue with [Warning: Prop className did not match.](https://github.com/vercel/next.js/issues/7322)